aesgcm/src/lib.rs

```diff
@@ -13,6 +13,9 @@ mod aes_gcm;
 mod aes_gcm_128;
 mod aes_gcm_256;
 
+use libcrux_traits::aead::{arrayref, consts, slice, typed_owned};
+
+// TODO: should this trait be re-exported here?
```

Comment:
Yes, we should always re-export the relevant crates so that users don't have to find the right crate with the trait.